core/src/main/java/io/questdb/cairo/arr/FlatArrayView.java

```diff
@@ -177,17 +177,21 @@ default int linearSearch(double value, int offset, int length) {
 
     /**
      * Computes the average of the block of elements in this flat array.
+     * Uses Kahan summation.
      *
      * @param offset the starting offset of the block (in elements)
      * @param length the number of elements in the block
      */
     default double sumDouble(int offset, int length) {
```

Comment:
There is override for `sumDouble` in `BorrowedFlatArrayView`, which also needed to be revised.